parts/linux/cloud-init/artifacts/ubuntu/mellanox_mlnx_ofed.list

```diff
@@ -0,0 +1,9 @@
+#
+# Mellanox Technologies Ltd. public repository configuration file.
+# For more information, refer to http://linux.mellanox.com
+#
+# For future reference:
+# https://network.nvidia.com/support/mlnx-ofed-public-repository/
+# https://www.mellanox.com/downloads/ofed/RPM-GPG-KEY-Mellanox
+# [mlnx_ofed_24.10-1.1.4.0_base]
+deb [signed-by=/etc/apt/keyrings/mellanox_mlnx_ofed.pub] http://linux.mellanox.com/public/repo/mlnx_ofed/24.10-1.1.4.0/ubuntu24.04/arm64 ./
```

Comment:
We don't expose / request user to whitelist this URL `linux.mellanox.com` before we start making this public we should ensure we augment this public list of outbound URL.

Reply:
my understanding is access linux.mellanox.com is only required when packer install the driver into the VHD, and no need to access this URL at bootstrap time or runtime